josephshambrook/js-2023
# 🚀 upgrade to Astro v2

- Astro v2 upgrades
  - Update all packages to latest versions
  - Migrate content to [Content Collections](https://docs.astro.build/en/guides/content-collections/)
  - Remove unnecessary remark plugins
  - Add scripts for using check cli command
- Small fixes
  - Revert Spotify component to load when page is loaded, rather than when component is visible, to improve UX on mobile
  - open footer links in new tab when they are external only (e.g. GitHub)
